.github/renovate.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "config:recommended",
+        ":semanticCommitTypeAll(chore)",
+        "helpers:pinGitHubActionDigests"
+    ],
+    "commitMessagePrefix": "deps: ",
+    "ignorePresets": [
+        ":semanticPrefixFixDepsChoreOthers"
+    ],
+    "ignorePaths": [
+        "docs/",
+        "data/",
+        ".github/"
+    ],
+    "prConcurrentLimit": 0,
+    "rebaseWhen": "behind-base-branch",
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": [
+        "type: process"
+    ],
+    "semanticCommits": "disabled",
```

Comment:
This seems in conflict with the other semantic commit settings

Reply:
My memory is rust, but I think this is because the other one is doing "chore" and this one uses "fix" or something by default...

Reply:
I had some confusions over the default semantic commits behavior. The default value is `auto` which auto-detects if the repo had semantic commits messages. Seems like `"semanticCommits": "disabled"` would disable the semantic commits entirely, so I changed it to `enabled` so all the commits WILL be semantic commits. ps: I think Kurtis is referring to `":semanticCommitTypeAll(chore)"` which prefixes all commits with `chore`; without it the default uses both `chore` and `fix`, but seems like the `semanticCommits` still needs to be enabled for that to work...

Reply:
Renovate is kinda iffy on config stuff sometimes -- sometimes the default configs change and it's hard to understand why. Looking at these two docs:
https://docs.renovatebot.com/semantic-commits/
https://docs.renovatebot.com/presets-default/#semanticcommits

I would say that we can use either -- so let's just pick your favorite and if it doesn't work we'll PR it to the other one.